datumaro/plugins/cvat_format/extractor.py

```diff
@@ -315,6 +316,17 @@ def _load_items(self, parsed):
         return parsed
 
 class CvatImporter(Importer):
+    @classmethod
+    def detect(cls, context: FormatDetectionContext) -> None:
+        annot_file = context.require_file('*.xml')
```

Comment:
Maybe, we can also check for the `/annotation/` dir.

Reply:
But there is no such directory in the CVAT format.

Reply:
It's not specified anywhere (like in COCO and Datumaro), but I don't see reasons to exclude this option. We decided some time ago we'll support the same layout with `images` and `annotations` dirs in such cases.

Reply:
But I don't think the extractor even supports this layout. It only looks for an `images` directory [in the same directory as the XML file](https://github.com/openvinotoolkit/datumaro/blob/v0.2/datumaro/plugins/cvat_format/extractor.py#L24).

And as far as I can see from CVAT's code, it never creates an `annotations` directory either.

Reply:
Okay, maybe we need to fix it everywhere then.